packages/docs/docs/rules/standard-tsconfig.md

```diff
@@ -6,8 +6,12 @@ Special case of the File Contents rule for typescript configs. Using a template
 
 ### Options
 
+- `file` (Optional)
+  - Name of the file. Defaults to `tsconfig.json`.
 - `generator` (Optional)
   - Function that can generate the config
+- `referenceTsConfigPath`
```

Comment:
Can we call this `tsconfigReferenceFile`? We use the word file for the filename elsewhere in the repo and I think this makes it easier grok.